src/main/java/nextstep/subway/line/application/SectionService.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.subway.line.application;
+
+import javax.transaction.Transactional;
+
+import org.springframework.stereotype.Service;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.domain.Section;
+import nextstep.subway.line.dto.SectionRequest;
+import nextstep.subway.station.application.StationService;
+
+/**
+ * @author : byungkyu
+ * @date : 2021/01/06
+ * @description :
+ **/
+@Service
+@Transactional
+public class SectionService {
+
+	private final LineService lineService;
+	private final StationService stationService;
+
+	public SectionService(LineService lineService, StationService stationService) {
+		this.lineService = lineService;
+		this.stationService = stationService;
+	}
+
+	public void addSection(Long lineId, SectionRequest sectionRequest) {
+		Line line = lineService.getLineById(lineId);
+		line.addSection(new Section(sectionRequest.getUpStationId(), sectionRequest.getDownStationId(),
```

Comment:
```suggestion
		line.addSection(sectionRequest.getUpStationId(), sectionRequest.getDownStationId(),
		...
```
section 객체 생성에 필요한 값만 전달해주면 코드가 좀더 간결해질 것 같네요